forelesning08/README.md

```diff
@@ -164,14 +164,18 @@ let workout = Workout()
 workout.entries = 14
 workout.name = "Pull-ups"
 
-let path = libDir.URLByAppendingPathComponent("workouts").path!
+let path = libDir.appendingPathComponent("workouts")
 
 // Serialisere til disk
-NSKeyedArchiver.archiveRootObject(workout, toFile: path)
+if let data = try? NSKeyedArchiver.archivedData(withRootObject: workout, requiringSecureCoding: true) {
+    try? data.write(to: path)
+}
 
 // Deserialisere fra disk
-let savedWorkout = NSKeyedUnarchiver.unarchiveObjectWithFile(path) as! Workout
-print("\(savedWorkout.name), entries: \(savedWorkout.entries)")
+if let archivedData = try? Data(contentsOf: path),
+        let savedWorkout = (try? NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(archivedData)) as? Workout {
+    print("\(savedWorkout.name), entries: \(savedWorkout.entries)")
+}
```

Comment:
`NSKeyedArchiver.archiveRootObject` and `NSKeyedUnarchiver.unarchiveObjectWithFile` are deprecated. I _think_ this is the equivalent, but haven't tested.